tests/test_plugin.py

```diff
@@ -33,3 +45,31 @@ def test_destroy(fxa_client, fxa_account):
     result = testdir.runpytest()
     result.assert_outcomes(error=1, passed=1)
     assert 'ClientError: Unknown account' in result.stdout.str()
+
+
+def test_commandline_email_option(testdir, monkeypatch, random_email):
+    monkeypatch.setenv('TEST_EMAIL', '{}'.format(random_email))
```

Comment:
We don't want to set the environment variable in this test, as this might not be testing the command line argument. Once removed, there's no need for `monkeypatch` or `import os` in this test.